src/jobs/schemas/job.schema.ts

```diff
@@ -83,45 +83,32 @@ export class JobClass extends OwnableClass {
   })
   statusHistory: Record<string, string>[];
 
-  // messages
+  // configuration version
   @ApiProperty({
-    type: Object,
+    type: String,
     required: false,
-    description:
-      "This is the equivalent object of the message sent to external service.",
+    description: "Stores the job's latest configuration version.",
   })
   @Prop({
-    type: Object,
+    type: String,
     required: false,
   })
-  messageSent: Record<string, unknown>;
+  configVersion: string;
 
-  // configuration definition (schema)
+  // messages
   @ApiProperty({
     type: Object,
     required: false,
     description:
-      "This is the equivalent object of the job configuration used to create this job.",
+      "This is the equivalent object of the message sent to external service.",
   })
   @Prop({
     type: Object,
     required: false,
   })
-  configuration: Record<string, unknown>;  // TBD
+  messageSent: Record<string, unknown>;
 
-  // parameters (instance)
-  @ApiProperty({
-    type: Object,
-    required: false,
-    description:
-      "This is the equivalent object of the jobs parameters provided by the user.",
-  })
-  @Prop({
-    type: Object,
-    required: false,
-  })
-  jobParams: Record<string, unknown>;
-  // TODO email address for owner from scicat? see create example for job.recipients
```

Comment:
We should keep this section because it is the only way to know which parameters the user sent with the create request.
...unless we have them somewhere else?

Reply:
True, this information is not stored anywhere else. I restored it